@here/harp-mapview/lib/MapView.ts

```diff
@@ -784,6 +785,7 @@ export class MapView extends THREE.EventDispatcher {
         this.m_camera.up.set(0, 0, 1);
         this.m_lookAtDistance = 0;
         this.m_focalLength = 0;
+        this.m_scene.userData = this;
```

Comment:
Object3D's userData is subject to JSON.stringify: https://github.com/mrdoob/three.js/blob/dev/src/core/Object3D.js#L749

in case of complex object it may throw circular reference (or similar) error.

Reply:
This is a left-over from previous solution, it should have been removed. Good catch ;)